build.sbt

```diff
@@ -3,11 +3,15 @@ import scala.sys.process._
 import sbt.Tests._
 
 val sparkVersion = "2.4.3"
-val scalaMajorMinor = "2.11"
 
-ThisBuild / scalaVersion := s"$scalaMajorMinor.12"
+lazy val scala212 = "2.12.8"
+lazy val scala211 = "2.11.12"
+lazy val supportedScalaVersions = List(scala212, scala211)
+
+ThisBuild / scalaVersion := scala211
 ThisBuild / organization := "io.projectglow"
 ThisBuild / scalastyleConfig := baseDirectory.value / "scalastyle-config.xml"
+ThisBuild / crossScalaVersions := Nil
```

Comment:
Hm, I would expect this to be scala211 and scala212. Could you please comment why it's like this?